addons/LDtk-Importer/LDtk.gd

```diff
@@ -73,30 +78,29 @@ func new_entity(entity_data, level, options):
 	else:
 		printerr("Could not load entity data: ", entity_data)
 		return
-
+	
 	if not new_entity:
 		return
-
+	
 	for data in metadata:
 		if data['name'] in new_entity:
 			new_entity[data['name']] = data['value']
 		else:
 			new_entity.set_meta(data['name'], data['value'])
-
+	
 	if is_custom_entity:
 		return new_entity
-
+	
```

Comment:
empty lines should be empty

Reply:
Should they really be empty here since it's the same indentation?

Reply:
Maybe not, not really that important. Do what you think is better.